Public/Src/Engine/Dll/EngineSchedule.cs

```diff
@@ -825,6 +826,11 @@ private bool ShouldSerializeOptimizationDataStructurePostExecution()
             }
         }
 
+        private static bool ShouldRemoveEmptyDirectories(IConfiguration configuration, string path)
+        {
+            return configuration.Schedule.UnsafeDisableSharedOpaqueEmptyDirectoryScrubbing && Directory.Exists(path) && !Directory.EnumerateFileSystemEntries(path).Any();

```

Comment:
>UnsafeDisableSharedOpaqueEmptyDirectoryScrubbing [](start = 42, length = 48)

`Directory.EnumerateFileSystemEntries(path)` is painfully slow. If perf. matters, then we need to look for another way to check directory contents; perhaps using FindFirstFile. Otherwise, you can leave a remark there mentioning this issue.

Reply:
The current usage of this flag is to be run in an atypical scenario, where BXL is called exactly once, and all directories checked will be empty -as far as I have tested-, so it should not be a bottleneck.

It doesn't seem worth to break compatibility just for this. I have added a comment in case anyone ever uses it.